### PR TITLE
Update README.MD with new links and deployment information

### PR DESCRIPTION
**Pull request by jillcary**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Add latest deployment info and links to the cli

**Checklist:**

- [x ] All tests passed
- [x] The code follows the code style and conventions of this project
- [x] The change requires a change to the documentation
- [x] I have updated the documentation accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,19 +96,90 @@ the Remote Monitoring WebApp
...
 1. Click on the link that is shown in the output window, it will take you to
    the Remote Monitoring WebApp
 1. It can take upto 5 minutes for the webapp to be ready
 1. Go to {azurewebitesurl}/hubmanager/v1/status to see HubManager microservice status
 1. Go to {azurewebitesurl}/devices/v1/status to see Devices microservice status
 
-## Local
-Please look [here](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Running-the-Remote-Monitoring-Solution-Locally) for more information for using this option
+### Local Deployment
+Please look [here](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy-local for more information for using this option
```
Comment (ppathan) [inline README.md]: The link is missing `)` at the end

File: README.md [hunk 2]
```diff
@@ -5,101 +5,76 @@
 Azure IoT PCS CLI Overview
 ==========================
 
-Command Line Interface for deploying an IoT preconfigured solution into a
-user's subscription.
+Command Line Interface for deploying an [Azure IoT solution](https://www.azureiotsolutions.com) into a
```
Comment (dluc) [inline README.md]: missing "accelerator"

File: README.md [hunk 3]
```diff
@@ -5,101 +5,76 @@
 Azure IoT PCS CLI Overview
 ==========================
 
-Command Line Interface for deploying an IoT preconfigured solution into a
-user's subscription.
+Command Line Interface for deploying an [Azure IoT solution](https://www.azureiotsolutions.com) into a
+user's Azure subscription.
 
-This CLI has the ability to deploy two configurations of PCS solutions:
+An IoT Solution is open source implementation of a common IoT solution patterns that you can deploy to Azure using your subscription. Each solution combines custom code and Azure services to implement a specific IoT scenario or scenarios. You can customize any of the scenarios to meet your specific requirements. Visit https://www.azureiotsolutions.com for more details or to deploy using the GUI.
```
Comment (dluc) [inline README.md]: * split line
* missing "accelerator"
* typo on "a common .. patterns"
* nit: in "... combine ... Azure services..." remove "Azure"
* nit: scenarios are not customizable, it's the architecture/solution that can be
* make url a clickable link

File: README.md [hunk 4]
```diff
@@ -5,101 +5,76 @@
...
 
-1. Basic - deploys all resources to a single VM.
-1. Standard - deploys resources using Azure Container Service and Kubernetes across multiple VMs.
-1. Local - deploys resources to be used for running and debugging microservices locally.
+### Features
 
-Dependencies
-============
-The command line interface depends on:
+This CLI has the ability to deploy the following configurations of PCS solutions:
```
Comment (dluc) [inline README.md]: drop "PCS" term

File: README.md [hunk 5]
```diff
@@ -5,101 +5,76 @@
...
-## Standard
+## 1. Prerequisites
 
-The Standard deployment offers a production ready deployment that can be
-scaled up or down as needed. It uses
-[Azure Container Service](https://azure.microsoft.com/en-us/services/container-service/)
-and [Kubernetes](https://kubernetes.io/) for orchestration. It would be nice to have installed
-[kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for running commands on kubernetes
-in addition to ```pcs```.
+* [nodejs](https://nodejs.org/en/) used as the runtime for the CLI.  Please install node before attempting a deployment.
```
Comment (dluc) [inline README.md]: in "https://nodejs.org/en/" remove "/en/"

File: README.md [hunk 6]
```diff
@@ -121,19 +96,90 @@ the Remote Monitoring WebApp
...
+## Overview
+
+When you deploy the preconfigured solution, there are several options that configure the deployment process:
+
+| Option | Values | Description |
+| ------ | ------ | ----------- |
+| SKU    | `basic`, `standard`, `local` | A _basic_ deployment is intended for test and demonstrations, it deploys all the microservices to a single virtual machine. A _standard_ deployment is intended for production, it deploys the microservices to multiple virtual machines. A _local_ deployment configures a Docker container to run the microservices on your local machine, and uses Azure services, such as storage and Cosmos DB, in the cloud. |
+| Runtime | `dotnet`, `java` | Selects the language implementation of the microservices. |
+
+To learn about how to use the local deployment, see [Running the remote monitoring solution locally](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Running-the-Remote-Monitoring-Solution-Locally#deploy-azure-services-and-set-environment-variables).
```
Comment (dluc) [inline README.md]: split long line

File: README.md [hunk 7]
```diff
@@ -121,19 +96,90 @@ the Remote Monitoring WebApp
...
+
+## Basic
+
+The purpose of the basic deployment is to demo the capabilities of the system
+and requires minimal setup, deploying all resources to a single VM.
+
+Creating a Basic solution will result in the following Azure services being
```
Comment (dluc) [inline README.md]: s/Creating/Deploying/

Comment (dluc) [reply README.md]: nit: need consistency, either "basic" and "standard" or "Basic" and "Standard" across the entire doc

File: README.md [hunk 8]
```diff
@@ -121,19 +96,90 @@ the Remote Monitoring WebApp
...
+provisioned into your Azure subscription:
+
+| Resource                       | Used For  |
+|--------------------------------|-----------|
+| [Linux Virtual Machine][virtual-machines]  | Hosting microservices |
+| [Azure IoT Hub][iot-hub]                   | Device management and communication |
+| [Azure Cosmos DB][cosmos-db]               | Storing configuration data, and device telemetry like rules, alerts, and messages |
+| [Azure Storage Account][storage-account]   | Storage for checkpoints |
+| [Azure Stream Analytics][stream-analytics] | Transforms data into messages and alerts |
```
Comment (dluc) [inline README.md]: s/Transforms data into messages and alerts/Processes and stores telemetry, and create alarms/

btw, do we use "alerts" or "alarms"?

File: README.md [hunk 9]
```diff
@@ -121,19 +96,90 @@ the Remote Monitoring WebApp
...
+and requires minimal setup, deploying all resources to a single VM.
+
+Creating a Basic solution will result in the following Azure services being
+provisioned into your Azure subscription:
+
+| Resource                       | Used For  |
+|--------------------------------|-----------|
+| [Linux Virtual Machine][virtual-machines]  | Hosting microservices |
+| [Azure IoT Hub][iot-hub]                   | Device management and communication |
+| [Azure Cosmos DB][cosmos-db]               | Storing configuration data, and device telemetry like rules, alerts, and messages |
```
Comment (dluc) [inline README.md]: nit: "Storing" or "Stores"

File: README.md (hunk 8 above)
Comment (jillcary) [reply README.md]: "alerts" is the preferred terminology.

**Pull request merged**